examples/direct_fidelity_estimation.py

```diff
@@ -327,15 +328,15 @@ def main(*, n_trials: int, n_clifford_trials: int, samples_per_term: int):
 
     noise = cirq.ConstantQubitNoiseModel(cirq.depolarize(0.1))
     print('Noise model: %s' % (noise))
+    noisy_simulator = cirq.DensityMatrixSimulator(noise=noise)
 
     estimated_fidelity = direct_fidelity_estimation(
         circuit,
         qubits,
-        noise,
+        noisy_simulator,
         n_trials=n_trials,
         n_clifford_trials=n_clifford_trials,
         samples_per_term=samples_per_term)
-    print('Estimated fidelity: %f' % (estimated_fidelity))
```

Comment:
You probably want to keep this. 

(Note that this one is different from the other printfs since it's in the main() function - outputing the estimated fidelity is the objective of this script.)

Reply:
Done.